keyboards/clueboard/66/keymaps/muzfuz/keymap.c

```diff
@@ -0,0 +1,70 @@
+  ESC_CTL,_AL_KCA,  KC_S,     KC_D,    _FL_KCF,    KC_G,     KC_H,     _FL_KCJ,    KC_K,   KC_L,   KC_SCLN,KC_QUOT,KC_NUHS,KC_LGUI,                            \
+  KC_LSFT,KC_LSFT,  KC_Z,     KC_X,    KC_C,       KC_V,     KC_B,     KC_N,       KC_M,   KC_COMM,KC_DOT, KC_SLSH,KC_SLSH,KC_RSFT,        KC_UP,           \
+  CMD_ENT,          KC_LCTL,  KC_LALT,  KC_LGUI,          KC_ENT, KC_SPC,                        KC_RGUI,KC_RALT,KC_RCTL,MO(_CL), KC_LEFT,KC_DOWN,KC_RGHT),
+
+  /* Keymap _FL: Function Layer
+   */
+[_FL] = KEYMAP(
+  KC_ESC, KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,  KC_F10, KC_F11, KC_F12, _______,KC_DEL,         KC_VOLU, \
+  _______,HASHRKT,_______,KC_LCBR,KC_RCBR,_______,_______,KC_UNDS,KC_PLUS,KC_PIPE,_______,_______,_______,KC_MUTE,                KC_VOLD, \
+  _______,CLNEQLS,_______,KC_LBRC,KC_RBRC,_______,_______,KC_MINS,KC_EQL,KC_BSLS,KC_TILD,KC_GRV,_______,_______,                         \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        KC_PGUP,         \
+  _______,_______,_______,_______,        _______,_______,                        _______,_______,_______,_______,KC_HOME,KC_PGDN,KC_END),
+
+  /* Keymap _FL: Function Layer
+   */
+[_AL] = KEYMAP(
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______, \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,                _______, \
+  _______,_______,_______,_______,_______,_______,KC_LEFT,KC_DOWN,KC_UP  ,KC_RGHT,_______,_______,_______,_______,                         \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,         \
+  _______,_______,_______,_______,        _______,_______,                        _______,_______,_______,_______,_______,_______,_______),
+
+  /* Keymap _CL: Control layer
+   */
+[_CL] = KEYMAP(
+  BL_STEP,RGB_M_P,RGB_M_B,RGB_M_R,RGB_M_SW,RGB_M_SN,RGB_M_K,RGB_M_X,RGB_M_G,_______,_______,_______,_______,_______,RGB_TOG,        RGB_VAI, \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,RESET,                    RGB_VAD, \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,                         \
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        RGB_SAI,         \
+  _______,_______,_______,_______,        RGB_MOD,   RGB_MOD,                     _______,_______,_______,_______,RGB_HUD,RGB_SAD,RGB_HUI),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`. 

https://docs.qmk.fm/#/feature_macros